swan-lake/resources/featured-scenarios/write-a-restful-api-with-ballerina.md

```diff
@@ -28,76 +26,161 @@ This guide will walk you through creating a RESTful API with two endpoints. In t
 
 ### The first endpoint
 
-The first endpoint is about getting data from the service as well as adding data to the service. Therefore, the service should handle both HTTP `GET` and `POST` requests.
-- The `GET` request is to get data, and the response should be `200 OK`.
-- The `POST` request is to add data, and the response should be `201 Created`.
+The first endpoint is about getting data from the service and adding data to the service. Therefore, the service should handle HTTP `GET` and `POST` requests.
+- The `GET` request is to retrieve data; the response should be `200 OK`.
+- The `POST` request is to add data; the response should be `201 Created`.
 
 ### The second endpoint
 
-The second endpoint is about getting data filtered from the service. The data is filtered by the ISO code. Therefore, the second endpoint accepts the ISO code as part of the URL and responds with the `200 OK` status code. In the event of an error, the relevant error is sent back to the client.
+The second endpoint is about getting data filtered from the service. The data is filtered by the ISO code. Therefore, the second endpoint accepts the ISO code as part of the URL and responds with the `200 OK` status code. The relevant error is sent back to the client in the event of an error.
 
 >**Info:** For the complete source code of this implementation, see [The complete code](/learn/write-a-restful-api-with-ballerina/#the-complete-code).
 
-## Create the service package 
+## Create the package 
 
-Ballerina uses packages to group code. You need to create a Ballerina package and write the business logic in it. In the terminal, execute the command below to create the Ballerina package for the API implementation.
+Ballerina uses packages to group code. Follow the steps below to create a Ballerina package and write the business logic. 
 
 > **Info:** For more information on Ballerina packages, see [Organize Ballerina code](/learn/organize-ballerina-code/).
 
-```
-$ bal new covid19 -t service
-```
+1. In the terminal, execute the command below to create the Ballerina package for the API implementation.
 
-You view the output below.
+    ```
+    $ bal new write-a-restful-api
+    ```
 
-```
-Created new package 'covid19' at covid19.
-```
+    You should see the output similar to the following.
```

Comment:
We need `the` article here for the noun `output`. The explanation is as follows:

Yes, including "the" in the sentence makes it grammatically correct. So it would be: "You should see the output similar to the following." This usage indicates a specific output that the reader should expect to encounter.

Reply:
Changed to: `You should see an output similar to the following.`